cli/client.go

```diff
@@ -584,6 +593,17 @@ var clientRetrieveCmd = &cli.Command{
 			return fmt.Errorf("The received offer errored: %s", offer.Err)
 		}
 
+		if cctx.String("maxPrice") != "" {
+			maxPrice, err := types.ParseFIL(cctx.String("maxPrice"))
```

Comment:
I feel like we really should have some default (like 1FIL)

Reply:
Will do